test/integration/multinode_test.go

```diff
@@ -446,11 +447,20 @@ func validateDeployAppToMultiNode(ctx context.Context, t *testing.T, profile str
 			t.Errorf("Pod %s could not resolve 'kubernetes.default': %v", name, err)
 		}
 	}
+
 	// verify both pods could resolve to a local service.
 	for _, name := range podNames {
 		_, err = Run(t, exec.CommandContext(ctx, Target(), "kubectl", "-p", profile, "--", "exec", name, "--", "nslookup", "kubernetes.default.svc.cluster.local"))
 		if err != nil {
 			t.Errorf("Pod %s could not resolve local service (kubernetes.default.svc.cluster.local): %v", name, err)
 		}
 	}
+
+	// verify both pods could resolve "host.minikube.internal"
```

Comment:
Let's move this fun to its own validateFunc with a comment 
And let it be a subtest so in the auto generated list of test and gopogh it shows up

Reply:
@prezha so it would be like
```
// validatePodsPingHost ..... explain what test does
func validatePodsPingHost(ctx context.Context, t *testing.T, profile string) {

```

then run `make generate-docs`

Reply:
@medyagh ok, i've moved it to a separate validateFunc and subtest

Reply:
btw, i did run `make generate-docs` - worked when tested locally but the unit_test here still complains?